### ISSUE TITLE
SuiteP - Emails colums + import button issues 

### ISSUE THREAD

**Issue by Mausino** [salesagility/SuiteCRM#2326]
#### Issue

1. Import button on pop up window has old style
2. Email columns tooks only half of screen

![emial1](https://cloud.githubusercontent.com/assets/5773777/18877364/51589ef0-84cd-11e6-87c3-d793eb9a2613.png)
#### Steps to Reproduce

1. Emails / Listview
2. Import email into any record
#### Your Environment

- SuiteCRM Version used: Official SuiteCRM demo 7.7.4
- Browser name and version: Chrome 53.x m 32 bit
- Operating System and version: Windows 10

**Comment by horus68** [comment]
Confirmed in Hotfix branch too!

#### My Environment
* SuiteCRM Version used: 7.7.4
* Browser name and version: Firefox 49.0.1 (32-bit) / Chrome 53.0.2785.116 m
* Environment name and version (Wamp, PHP 5.5.12, Mysql 5.6.17):
* Operating System and version: Windows 10 Pro

**Comment by shogunpol** [comment]
The issue has been tested on recent version of SuiteCRM(7.7.4), and it appear,
We assign this issue as a bug.
@horus68 , thanks for additional informations.
@Mausino  , thanks for raise this issue.

**Comment by glidester** [comment]
I have a workaround (not a fix)

I had a user complaining that the column widths were collapsing on each refresh of the Emails page. They had to keep resizing the columns after each reload. 

After much head scratching I managed to figure out that the column widths are controlled by the cookie value 'EmailGridWidths'. I set the value to "0=10&1=10&2=350&3=650&4=125&5=250" which expanded the columns to take up most of the screen width.

The code that sets the cookie value seems to be very temperamental (width adjustments rarely seem to stick), I found multiple 'EmailGridWidths' values in the cookie with different expiry times. Perhaps this issue could also be looked at the same time as the OP issue.
